Test Failure: junit.framework.TestSuite.io.openliberty.microprofile.metrics.internal.fat.LibraryRefTest

RTC 302081

This is a Windows only issue.

```
io.openliberty.microprofile.metrics.internal.fat.LibraryRefTest:java.lang.RuntimeException: Error getting server NonExistentLibrary
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:206)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:99)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:81)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:58)
at componenttest.rules.repeater.FeatureReplacementAction.setFeatures(FeatureReplacementAction.java:672)
at componenttest.rules.repeater.FeatureReplacementAction.setup(FeatureReplacementAction.java:512)
at componenttest.rules.repeater.JakartaEEAction.setup(JakartaEEAction.java:236)
at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:148)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:374)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
Caused by: componenttest.exception.TopologyException: (On Repeat Action _EE11_FEATURES_MicroProfile_70) Unable to delete pre-existing server directory: C:/Users/jazz_build/Build/workspace/ebcTestRunner/dev/autoFVT/image/output/wlp/usr/servers/NonExistentLibrary
at componenttest.topology.impl.LibertyServer.setup(LibertyServer.java:880)
at componenttest.topology.impl.LibertyServer.<init>(LibertyServer.java:820)
at componenttest.topology.impl.LibertyServerFactoryDelegate.newLibertyServer(LibertyServerFactoryDelegate.java:38)
at componenttest.topology.impl.LibertyServerFactoryDelegate.createLibertyServer(LibertyServerFactoryDelegate.java:33)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:162)
```

--- a/dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/src/io/openliberty/microprofile/metrics/internal/fat/LibraryRefTest.java
+++ b/dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/src/io/openliberty/microprofile/metrics/internal/fat/LibraryRefTest.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2023, 2024 IBM Corporation and others.
+ * Copyright (c) 2023, 2025 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
@@ -129,6 +129,7 @@ public class LibraryRefTest {
 	public void nonExistentLibrary() throws Exception {
 		server = serverNonExistentLibrary;
 		server.startServer();
+		server.waitForSSLStart();
 
 		// CWWKG0033W The value [<value>] specified for the reference attribute
 		// [libraryRef] was not found in the configuration.
@@ -216,6 +217,7 @@ public class LibraryRefTest {
 		}
 
 		server.startServer();
+		server.waitForSSLStart();
 
 		// CWMMC0014I emits that metrics is using libraryRef
 		Assert.assertNotNull("CWMMC0014I Not found",
@@ -276,6 +278,7 @@ public class LibraryRefTest {
 	public void externalMicrometerUselessJar() throws Exception {
 		server = serverMicrometerUseless;
 		server.startServer();
+		server.waitForSSLStart();
 
 		// CWMMC0014I emits that metrics is using libraryRef
 		Assert.assertNotNull("CWMMC0014I Not found",

